recipes/pcapplusplus/all/conanfile.py

```diff
@@ -13,20 +15,24 @@ class PcapplusplusConan(ConanFile):
     homepage = "https://github.com/seladb/PcapPlusPlus"
     settings = "os", "compiler", "build_type", "arch"
     options = {
+        "fPIC": [True, False],
         "immediate_mode": [True, False],
     }
     default_options = {
+        "fPIC": True,
         "immediate_mode": False,
     }
+
+    exports_sources = "patches/*"
     generators = "make"
 
-    _source_subfolder = "PcapPlusPlus"
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
 
-    def configure(self):
-        if self.settings.os not in ["Macos", "Linux"]:
-            raise ConanInvalidConfiguration("%s is not supported" % self.settings.os)
-
-        del self.settings.compiler.libcxx
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
Windows is currently not supported because WinPcap is not supported in ConanCentral

Reply:
Indeed, but hopefully winpcap will be available soon.
These extra lines don't cost much extra effort.